### PR TITLE
Home screen changes to r entals

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs [hunk 1]
```diff
@@ -402,14 +402,12 @@ rentalBannerConfig state =
...
-      , imageHeight = V 43
+      , imageHeight = V 45
       , imageWidth = V 66
       , imagePadding = PaddingVertical 0 0
-      , title = getString RIDE_BOOKING <> (maybe "" (\rentalsInfo ->
-                                                      let timeUTC = rentalsInfo.rideScheduledAtUTC
-                                                          currentUTC = EHC.getCurrentUTC ""
-                                                          date = if EHC.convertUTCtoISC timeUTC "Do" == EHC.convertUTCtoISC currentUTC "Do" then "at " else "on " <>  EHC.convertUTCtoISC timeUTC "ddd" <> ", "
-                                                      in date <> EHC.convertUTCtoISC timeUTC "hh" <> ":" <> EHC.convertUTCtoISC timeUTC "mm" <> " " <> EHC.convertUTCtoISC timeUTC "A"
+      , title = "You have an upcoming rental booking " <> (maybe "" (\rentalsInfo ->
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs:408]: Add translations

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs [hunk 2]
```diff
@@ -443,3 +488,92 @@ type ServiceabilityResponse = {
   , geoJson :: String
   , specialLocCategory :: String
 }
+
+dummyFareQuoteDetails = {
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs:492]: Please add type signatures

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 3]
```diff
@@ -3554,45 +3439,41 @@ pickupLocationView push state =
...
-                  , gravity BOTTOM
+          , linearLayout[
+              height WRAP_CONTENT
+            , width MATCH_PARENT
+            , padding $ Padding 16 8 16 16
+            ][  linearLayout
+                [ height WRAP_CONTENT
+                , width MATCH_PARENT 
+                , padding $ Padding 14 16 14 16 
+                , stroke $ "1,#373B49"
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:3450]: Add this to color file

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs [hunk 4]
```diff
@@ -443,3 +488,92 @@ type ServiceabilityResponse = {
...
+                in (transFormQuoteDetails quoteDetails)
+              _ -> dummyFareQuoteDetails
+          _  -> dummyFareQuoteDetails
+    in { quoteDetails : quoteDetails, index : index, activeIndex : 0 , fareDetails : fareDetails}
+    ) sortedByFare)
+  continue state { data{quotesList = rentalsQuoteList, selectedQuote = if MB.isNothing state.data.selectedQuote then DA.head $ rentalsQuoteList else state.data.selectedQuote}}
+  where 
+    transFormQuoteDetails quoteDetails = 
+        { includedKmPerHr : MB.fromMaybe 0 quoteDetails.includedKmPerHr 
+        , nightShiftCharge : MB.fromMaybe 250 quoteDetails.nightShiftCharge
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs:564]: why 250?

**Review by aryan-13 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs (hunk 4 above)
Comment (MercyQueen) [reply Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs:564]: this is the default value that we are have been using

**Pull request merged**